independent-publisher-connectors/WebEx/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "C5dd1ba47c91e7a80dd71a82f465918bf0e7b9ff260162f623ef583a06123c641",
+          "scopes": [
+            "meeting:schedules_read meeting:schedules_write meeting:preferences_read meeting:preferences_write meeting:controls_read  meeting:controls_write meeting:participants_read meeting:participants_write meeting:transcripts_read spark:all"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://webexapis.com/v1/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://webexapis.com/v1/access_token"
+            },
+            "refreshUrl": {
+              "value": "https://global.consent.azure-apim.net/redirect"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "University College London"
```

Comment:
@ocarhui please add the stackOwner property.
"stackOwner": "Name of company or organization that connector is connecting to."